### PR TITLE
fix: add explanation for php document

### PR DESCRIPTION
**Pull request by wangggong**
Add some explanation for phpactor server and its prerequisites in `modules/lang/php/README.org`.

### CODE REVIEW

File: modules/lang/php/README.org [hunk 1]
```diff
@@ -65,7 +64,11 @@ This module has no dedicated maintainers.
 
 * Prerequisites
 ** PHP
-To get started with PHP, you'll need ~php~ (5.3+) and ~composer~:
+To get started with PHP, you'll need ~php~ (5.3+) and ~composer~.
+
+Note for =+lsp=:
+1. In order to make full use of phpactor server, ~php~ (7.3+) is recommended.
+2. If you use intelephense, ~node~ and ~npm~ is needed.
```
Comment (hlissner) [inline modules/lang/php/README.org:71]: `s/is/are`

**Review by hlissner — CHANGES_REQUESTED** [review]
Looks good to me, just one small correction.

File: modules/lang/php/README.org (hunk 1 above)
Comment (wangggong) [reply modules/lang/php/README.org]: done

**Review by hlissner — APPROVED** [review]

**Comment by hlissner** [comment]
Thanks!

**Pull request merged**